packages/@okta/vuepress-site/docs/guides/build-sso-integration/specify-your-settings/openidconnect/settings.md

```diff
@@ -11,10 +11,11 @@
     * **Refresh token** &mdash; not supported for OIN app integrations
     * Optional. **Implicit (Hybrid)**
   * SPA:
+    * **Authorization code**
```

Comment:
is the c lower case here because you are mirroring the UI?

Reply:
Yes